romana-install/roles/provision/create-aws-hosts/templates/aws_inventory

```diff
@@ -0,0 +1,17 @@
+{% for i in stackinfo %}
+{{ i.Name }} ansible_ssh_host="{{ i.IP }}" ansible_ssh_user="ubuntu" ansible_ssh_private_key_file="{{ romana_ssh_key | realpath }}" stack_host="{{ i.Host }}"
+{% endfor %}
+
+[stack_nodes:children]
+controller
+computes
+
+[controller]
+{% for i in stackinfo %}{% if i.Role == "Controller" %}
+{{ i.Name }}
```

Comment:
It's more of a hostname or label. It's an ansible inventory, not a /etc/hosts file.